libs/vkd3d/vkd3d_private.h

```diff
@@ -1158,7 +1169,10 @@ struct d3d12_deferred_descriptor_set_update
 
 struct vkd3d_dynamic_state
 {
+    uint32_t active_flags; /* vkd3d_dynamic_state_flags */
     uint32_t dirty_flags; /* vkd3d_dynamic_state_flags */
+    uint32_t dirty_vbos;
+    uint32_t dirty_vbo_strides;
```

Comment:
Does this mask do anything that couldn't be handled by `dirty_vbos`? We have to rebind the vertex buffers anyway in case the stride changed, and we already invalidate the pipeline and bind time if necessary.

Reply:
Yes, this is relevant when we bind a pipeline which goes from being static stride to dynamic stride. In this case, strides are invalidated, but not the VBOs themselves. However, if we are using dynamic strides, we use the vbo | stride mask, and with static stride, only vbo mask.